bucket/makehuman.json

```
@@ -5,13 +5,18 @@
     "license": "AGPL-3.0-or-later",
     "url": "http://download.tuxfamily.org/makehuman/releases/makehuman-community-1.2.0-windows.zip",
     "hash": "7f801801f4307a89c8dc91e5713553000a37d1225f25cb1c32afad20f3b0f4ee",
-    "bin": "makehuman.exe",
-    "shortcuts": [
-        [
-            "makehuman.exe",
+    "bin": "makehuman.ps1",
+    "shortcuts": [[
```

```suggestion
    "shortcuts": [
        [
            "Python\\pythonw.exe",
            "MakeHuman",
            "$dir\\mhstartwrapper.py",
            "makehuman.ico"
        ]
    ],
```

**.ps1** works fine for bin. 
However for *shortcuts* there will be a problem if the user does not associate **.ps1** with `powershell.exe` (e.g. some user may associate **.ps1** with text editors such as `vscode` or `Powershell ISE`.

We can assign **shortcut icon** in the 4th column of *shortcut*.